src/scripts/router.coffee

```diff
@@ -52,8 +52,7 @@ define (require) ->
         # Sending analytics will be done once the path is canonicalized
 
       @route /^donate\/?([^/\?;]*)?\/?([^/\?;]*)?\/?([^/\?;]*)?(?:\?)?.*/, 'donate', (page, uuid, type) ->
-        @appView.render('donate', {page: page, uuid: uuid, type: type})
-        analytics.sendAnalytics()
+        window.location = 'https://openstax.org/give'
```

Comment:
Is this just to handle the case when someone lands directly on the Donate Page (e.g. from a bookmark or from a search engine)? If so, nice!